pandas/core/window/rolling.py

```diff
@@ -395,6 +395,8 @@ def _get_window_indexer(self, index_as_array):
         -------
         VariableWindowIndexer or FixedWindowIndexer
         """
+        if isinstance(self.window, BaseIndexer):
```

Comment:
so we ignore the arg when using a custom indexer? or should it still be passed (and can be ignored if needed inside the custom indexer?

imagine writing VariableIWindowIndexer and FixedWindowIndwxer as a BaseIndexer (which i think we should do at some point)

Reply:
All custom indexers should subclass `BaseIndexer` so `self.window` should be the custom indexer which will accept arguments from mean (`_apply`).

VariableWindowIndexer and FixedWindowIndexer already subclass `BaseIndexer`